flow-typed/glamor.js

```diff
@@ -5,3 +5,13 @@ declare module 'glamor' {
     },
   };
 }
+
+declare module 'glamor/react' {
+  declare module.exports: {
+    createElement: any,
+    dom: any,
+    vars: any,
+    makeTheme: any,
+    propMerge: Function,
+  };
```

Comment:
I wouldn't expect to see anything here beyond `createElement`. Are these others necessary?

Reply:
I was actually just looking at the Typescript typings available in glamor: https://github.com/threepointone/glamor/blob/master/react.d.ts.